modules/module-info.php

```diff
@@ -745,7 +745,14 @@ function jetpack_likes_more_info() { ?>
 
 	<p><?php esc_html_e( 'Likes allow your readers to show their appreciation for your posts and other published content using their WordPress.com accounts. Your readers will then be able to review their liked posts from WordPress.com.', 'jetpack' ) ?></p>
 	<p><?php esc_html_e( 'Displayed below your posts will be how many people have liked your posts and the Gravatars of those who have liked them.', 'jetpack' ); ?></p>
-
+	<?php if ( current_user_can( 'jetpack_manage_modules' ) ) : ?>
```

Comment:
This may not be necessary since they don't have access to the settings page at all if `! current_user_can( 'jetpack_manage_modules' )`, and it's not displayed on the main admin page.